src/main/java/seedu/address/logic/commands/FindContactCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.ContactContainsKeywordPredicate;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindContactCommand extends Command {
+
+    public static final String COMMAND_WORD = "findc";
```

Comment:
I thought that we previously agreed to use `findn` because `n` is a prefix for name when we add a new contact. Perhaps the Command can be `FindContactNameCommand` or `FindNameCommand` (I prefer the latter as we did not standardise calling a the client, "contact".

Reply:
ok sure